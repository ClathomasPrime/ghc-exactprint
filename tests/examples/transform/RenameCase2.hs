module RenameCase2 where

foo x = case (baz x) of
    1 -> "a"
    _ -> "b"

