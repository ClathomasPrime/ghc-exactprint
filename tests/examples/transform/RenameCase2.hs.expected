module RenameCase2 where

fooLonger x = case (baz x) of
    1 -> "a"
    _ -> "b"

